### PR TITLE
Feature/linter

### PR DESCRIPTION
**Pull request by wovalle**
Closes https://github.com/werewolfrd/werewolf-moderator/issues/21

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,13 +27,19 @@
...
     "json-loader": "^0.5.4",
-    "react": "^15.4.1",
-    "react-dom": "^15.4.1",
     "webpack": "^2.2.1",
     "webpack-dev-server": "^2.3.0"
   },
   "dependencies": {
+    "prop-types": "^15.5.8",
+    "react": "^15.4.1",
+    "react-dom": "^15.4.1",
```
Comment (mamodom) [inline package.json:42]: Why are `react` and `react-dom` dev-dependencies?

**Review by mamodom — COMMENTED** [review]
I now realized this diff is too hard to review since you renamed all `js` to `jsx`.

File: package.json (hunk 1 above)
Comment (wovalle) [reply package.json:42]: They were dev-dependencies, I moved them to dependencies.

Comment (wovalle) [reply package.json:42]: And yes @mamodom, it's quite difficult because of the naming but that's part of the linting process.

Comment (mamodom) [reply package.json:42]: Oh yeah you're right

**Review by vetom — APPROVED** [review]

**Comment by vetom** [comment]
So many conflicts tho. :full_moon_with_face:

**Comment by wovalle** [comment]
This branch will be abandoned because we moved to redux and started working in the linter in `feature/linter-v2`. Will close now.

**Pull request closed without merging**